trillian/ctfe/serialize.go

```diff
@@ -34,24 +37,25 @@ func (c *LogContext) signV1TreeHead(signer *crypto.Signer, sth *ct.SignedTreeHea
 		return nil
 	}
 
-	signature, err := signer.Sign(sthBytes)
+	h := sha256.New()
+	h.Write(sthBytes)
```

Comment:
`h := sha256.Sum256(sthBytes)` would be more concise.

Reply:
done

Reply:
It looks like you changed a similar bit of code in `buildV1SCT()`, but not this bit.